devconf-outreachy/source/index.rst

```diff
@@ -0,0 +1,415 @@
+  source cloud management and infrastructure software.
+  
+  The first reference offers a bit of OpenStack history.
+
+  If we look at Stacklytics for the current release cycle (Stein), the basic
+  stats can be seen on the slide.
+
+About Outreachy
+---------------
+
+What is Outreachy?
+
+  * Volunteer organization under Software Freedom Conservancy
+  * Brings together interns, mentors, and code
+  * Promotes diversity and free software
+
+How does it work?
+
+  * Interns get on board open source projects
+  * Each intern gets a mentor (or two)
+  * 3 month paid internship
+
+.. Things to talk about ^ (ietingof)
+
+  Outreachy is a volunteer organization operating under the Software Freedom
+  Conservancy umbrella.
+
+  Outreachy helps bring together interns, mentors, and open source projects
+  that seek contributors and mentors.
+
+  The main focus of Outreachy is to promote and ensure diversity, getting
+  different people from all over the world to join the open source movement.
+
+Why become an intern?
+---------------------
+
+* To find your aspirations
+* To learn the software engineering craft
+* To start contributing to free and open source software
+* To get a job (hopefully, the dream one)
+
+.. Things to talk about ^ (Aija)
+
+  There might be many reasons for young people to consider Outreachy
+  internship.
+
+  Taking part in large-scale software development could help a person to
+  understand if a software engineering career would be a good fit for them.
+
+  For greenhorn engineers, being on a team with masters of craft works like a
+  medieval apprenticeship and helps interns to grow.
+
+  For people with some background in software engineering, this can help
+  to get started contributing to free and open source software which they
+  have not yet tried for a variety of reasons.
+
+  Finally, working on a highly visible open source project helps build a
+  digital trace to prove one's worthiness to the prospective employers.
+
+Why mentor with @Outreachy?
+---------------------------
+
+* To contribute back to the culture we are part of
+* Grow professionally and personally
+* Get more hands on our stuff
+
+.. Things to talk about ^ (dtantsur)
+
+  One reason for being a mentor is to contribute back to the community which shaped
+  us - the grown up programmers.
+
+  We are the products of this culture, and need to sustain it to stay happy.
+
+  Meeting new and diverse people help us to improve personally.
+
+  Teaching can be quite rewarding - we learn new things when we explain them.
+
+  Finally, Outreachy brings motivated and productive contributors to our projects.
+
+Aija's internship @OpenStack
+----------------------------
+
+* How it all began
+
+  - Finding a project
+
+     - something new, but in familiar programming language
+     - mentors in the same or nearby time zone
+
+* The initial application:
+
+  - Challenges
+
+    - initial ticket
+    - transparency of other applying
```

Comment:
Change to "- transparency in seeing other applicant"

Reply:
Done